src/createNumbersGenerator.js

```diff
@@ -25,8 +25,15 @@
  *
  * @return {number}
  */
-function createNumbersGenerator(numbers, min, max) {
-  // write code here
+function createNumbersGenerator(
+  numbers = [],
+  min = -Infinity,
+  max = +Infinity) {
```

Comment:
write these as default values

Reply:
function createNumbersGenerator(numbers = [], min = -Infinity, max = +Infinity) {
  //some code
}
I wrote this params with new line, because linter return to me an error: